tests/test_defaults.py

```diff
@@ -109,6 +114,28 @@ def test():
             ('method', 'GET'), ('path', '/skip'), ('status', 200)
         )
 
+    def test_exception_counter_metric(self):
+        self.metrics(export_defaults=False)
+
+        @self.app.route('/error')
+        def test():
+            raise RuntimeError
+
+        self.client.get('/error')
+
+        self.assertMetric(
+            'flask_http_request_exceptions', '0.0',
+            ('method', 'GET'), ('path', '/error'), ('status', 200)
```

Comment:
Isn't this supposed to be 1 here?

Reply:
yupp!

Reply:
I'm a bit confused about this test. It doesn't export the defaults but we do need them (you added a new default metric).
Also, this still needs to say `1.0` because we have already requested the error endpoint.